backend/btrixcloud/orgs.py

```diff
@@ -1528,6 +1562,12 @@ async def set_role(
 
         return {"updated": True}
 
+    @router.post(
+        "/recalculate-storage", tags=["organizations"], response_model=SuccessResponse
+    )
+    async def recalculate_org_storage(org: Organization = Depends(org_owner_dep)):
+        return await ops.recalculate_storage(org)
```

Comment:
At first I thought maybe it should return the recalculated storage, but that would make it harder to convert to a bg job, so this makes sense.